frontend/server/src/DAO/Users.php

```diff
@@ -346,4 +399,43 @@ public static function getUsersCount(): int {
         /** @var int */
         return \OmegaUp\MySQLConnection::getInstance()->GetOne($sql);
     }
+
+    final public static function generateDeletionToken(
+        \OmegaUp\DAO\VO\Users $user,
+        string $token
+    ): void {
+        $sql = '
+            UPDATE
+                Users u
+            SET
+                u.deletion_token = ?
+            WHERE
+                u.user_id = ?;
+        ';
+        \OmegaUp\MySQLConnection::getInstance()->Execute(
+            $sql,
+            [$token, $user->user_id]
+        );
+    }
+
+    final public static function validateDeletionToken(
+        \OmegaUp\DAO\VO\Users $user,
+        string $token
+    ): bool {
+        $sql = 'SELECT
+                    COUNT(*)
+                FROM
+                    Users
+                WHERE
+                    `user_id` = ?
+                    AND `deletion_token` = ?;';
+
+        /** @var int */
+        $count = \OmegaUp\MySQLConnection::getInstance()->GetRow(
```

Comment:
If you want to get a single value, you can use tis function instead
```suggestion
        $count = \OmegaUp\MySQLConnection::getInstance()->GetOne(
```